src/openstudio_lib/OSCollapsibleItemList.cpp

```diff
@@ -167,6 +168,12 @@ void OSCollapsibleItemList::addCollapsibleItem(OSCollapsibleItem* collapsibleIte
   connect(collapsibleItem, &OSCollapsibleItem::openLibDlgClicked, this, &OSCollapsibleItemList::openLibDlgClicked);
 }
 
+void OSCollapsibleItemList::addCategoryPlaceholderItem(OSCategoryPlaceholder* categoryPlaceholderItem) {
+
+  m_placeholderItems.push_back(categoryPlaceholderItem);
+  m_vLayout->insertWidget(0, categoryPlaceholderItem);
+}
```

Comment:
@macumber This makes me realize we have a `std::vector<OSCollapsibleItem*> m_collapsibleItems;` but our `void OSCollapsibleItemList::addCollapsibleItem(OSCollapsibleItem* collapsibleItem)` does not add to it... even though other functions loop on it...

Reply:
Hmm, yeah looks like we never add anything to `m_collapsibleItems` so it's kind of useless.